src/screens/settings-test.tsx

```diff
@@ -398,6 +400,24 @@ export const SettingsTestScreen = () => {
         }}>
         {getUserAgent()}
       </Text>
+      <Spacer height={8} />
+      <Title text="Backend" />
+      <Text t11>{backend}</Text>
+      <Spacer height={8} />
+      <Button
+        title="Select backend"
+        onPress={async () => {
```

Comment:
Do not create functions during render, please exclude it into separate function with `useCallback`